### ISSUE TITLE
IN GENERAL: I've lost the overview of what is done and what not!

### ISSUE THREAD

**Issue by jojo12** [Heklaterriol/JEM-Project#126]
IN GENERAL: I've lost the overview of what is done and what not!
Please give a new version as base!

**Comment by Heklaterriol** [comment]
The given version from the developer is 2.3.9 and we created a branch "small-fixes-before-JEM4" for all the changes/fix from our side.
The developer is at work with our bugslist "fixing in round 5 (2.3.9)" for a next test release
Will be published here when ready.

**Comment by jojo12** [comment]
Thanks! Sorry!

**Comment by Heklaterriol** [comment]
Welcome and no problem.
Thank you for your contributions, support and questions.

**Comment by mckillo** [comment]
I agree. Define a date, so I'll try to close any issue more and create 2.3.10 (beta)

**Comment by jojo12** [comment]
@Heklaterriol : what do you mean?

**Comment by mckillo** [comment]
I mean, you can define a end date of this round to close maximum issues, and so, Hekla can create a new version. For example, try to close issues before next friday or tomorrw night.
